phpunit-bootstrap.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Given that this is more of a config file for the test suite, can you add this file to the config directory?